### PR TITLE
Bug/csi 1888 k8s code generation fail

### PR DESCRIPTION
**Pull request by matancarmeli7**

### CODE REVIEW

File: cmd/manager/main.go [hunk 1]
```diff
@@ -29,15 +29,18 @@ import (
 	"github.com/IBM/ibm-block-csi-operator/pkg/apis"
 	operatorConfig "github.com/IBM/ibm-block-csi-operator/pkg/config"
 	"github.com/IBM/ibm-block-csi-operator/pkg/controller"
+	operatorVersion "github.com/IBM/ibm-block-csi-operator/version"
```
Comment (oriyarde) [inline cmd/manager/main.go:32]: please merge latest develop into your private branch

File: cmd/manager/main.go [hunk 2]
```diff
@@ -136,34 +140,34 @@ func main() {
 	}
 
 	/*
-	// NOTE: Should enable it back in later version - issue CSI-613
```
Comment (oriyarde) [inline cmd/manager/main.go:141]: is this diff really necessary?

File: go.mod [hunk 3]
```diff
@@ -4,31 +4,31 @@ go 1.13
 
 require (
 	github.com/go-logr/logr v0.1.0
-	github.com/go-openapi/spec v0.17.2
```
Comment (oriyarde) [inline go.mod:7]: what about go.sum and vendor/?
is there any change to update in the auto-generated code?

File: cmd/manager/main.go [hunk 4]
```diff
@@ -30,15 +30,18 @@ import (
 	operatorConfig "github.com/IBM/ibm-block-csi-operator/pkg/config"
 	operatorVersion "github.com/IBM/ibm-block-csi-operator/version"
 	"github.com/IBM/ibm-block-csi-operator/pkg/controller"
+	operatorVersion "github.com/IBM/ibm-block-csi-operator/version"
```
Comment (oriyarde) [inline cmd/manager/main.go:33]: it seems the merge didn't remove this, is this diff really necessary?

**Review by zingero — CHANGES_REQUESTED** [review]
I agree with Ori

**Review by oriyarde — APPROVED** [review]
LGTM

**Review by zingero — APPROVED** [review]

**Pull request merged**